pkg/okta/authentication_middleware_test.go

```diff
@@ -68,7 +68,7 @@ func (tjv TestJwtVerifier) VerifyAccessToken(jwt string) (*jwtverifier.Jwt, erro
 	return tjv.verify(jwt)
 }
 
-func (s *AuthenticationMiddlewareTestSuite) buildMiddleware(verify func(jwt string) (*jwtverifier.Jwt, error)) func(http.Handler) http.Handler {
+func (s *AuthenticationMiddlewareTestSuite) buidMiddleWareFactory(verify func(jwt string) (*jwtverifier.Jwt, error)) *MiddlewareFactory {
```

Comment:
```suggestion
func (s *AuthenticationMiddlewareTestSuite) buildMiddleWareFactory(verify func(jwt string) (*jwtverifier.Jwt, error)) *MiddlewareFactory {
```

(Don't commit this directly, as it'll only rename the definition, not all the places it's used)